tox.ini

```diff
@@ -7,22 +7,14 @@ isolated_build = True
 
 [testenv]
 deps =
-    test: pytest
-    test: pytest-cov
-    test: pytest-mpi>=0.2
-
-    # For --pre, 2.0.0b1 has a header size difference that has been reverted, so avoid it
-    py39-deps: numpy>=1.19.3,!=2.0.0b1
-    py310-deps: numpy>=1.21.3,!=2.0.0b1
-    py311-deps: numpy>=1.23.2,!=2.0.0b1
-    py312-deps: numpy>=1.26.0,!=2.0.0b1
```

Comment:
this part is not replaced, but simply removed as not needed anymore: numpy 2.0.0b2 is long superseded and wouldn't be resolved to in any env anyway.